include/ignition/gazebo/gui/GuiEvents.hh

```diff
@@ -25,6 +25,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <sdf/Element.hh>
```

Comment:
alphabetize

Reply:
added line to separate the system headers and ignition / sdf headers and arranged them in alphabetical order. 9f4a753763f64918d2441b56f16cb8be9040c699